src/lib.rs

```diff
@@ -134,6 +142,10 @@ mod util;
 ///     `Some(...)`, relieving the caller from having to do this. Note that with this setting on
 ///     one cannot set the field to `None` with the setter - so the only way to get it to be `None`
 ///     is by using `#[builder(default)]` and not calling the field's setter.
+///
+///   - `transform = |params: Types, ...| expr`: this makes the setter accept `params: Types, ...`
```

Comment:
This is a weird way to document that `transform` supports multiple parameters. How about `|param1: Type1, param2: Type2, ...|`?